gammapy/cube/tests/test_cube_pipe.py

```diff
@@ -0,0 +1,121 @@
+"""Example how to make a Cube analysis from a 2D background model.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.tests.helper import assert_quantity_allclose
+from astropy.coordinates import SkyCoord, Angle
+from astropy.units import Quantity
+from ...data import DataStore
+from ...utils.testing import requires_dependency, requires_data
+from ...image import SkyMask
+from .. import StackedObsCubeMaker
+from ...background import OffDataBackgroundMaker
+from .. import SkyCube
+from ...utils.energy import Energy
+
+
+def make_empty_cube(image_size, energy, center, data_unit=None):
+    """
+    Make an empty `SkyCube` from a given `SkyMask` and an energy binning.
+
+    Parameters
+    ----------
+    image_size:int, Total number of pixel of the 2D map
+    energy: Tuple for the energy axis: (Emin,Emax,nbins)
+    center: SkyCoord of the source
+    data_unit : str, Data unit.
+    """
+    def_image = dict()
+    def_image["nxpix"] = image_size
+    def_image["nypix"] = image_size
+    def_image["binsz"] = 0.02
+    def_image["xref"] = center.galactic.l.deg
+    def_image["yref"] = center.galactic.b.deg
+    def_image["proj"] = 'TAN'
+    def_image["coordsys"] = 'GAL'
+    def_image["unit"] = data_unit
+    e_min, e_max, nbins = energy
+    empty_cube = SkyCube.empty(emin=e_min.value, emax=e_max.value, enumbins=nbins, eunit=e_min.unit, mode='edges',
+                               **def_image)
+    return empty_cube
+
+
+@requires_dependency('reproject')
+@requires_data('gammapy-extra')
+def test_cube_pipe(tmpdir):
+    tmpdir = str(tmpdir)
+    from subprocess import call
+    outdir = tmpdir
+    outdir2 = outdir + '/background'
+
+    cmd = 'mkdir -p {}'.format(outdir2)
```

Comment:
Could you use the method `tmpdir.mkdir()` and remove the subprocess call and print statement here?